docs/EigenPods.md

```diff
@@ -30,30 +34,40 @@ The following sections are all related to managing Consensus Layer (CL) and Exec
+### Effective Restaked Balance - Hysteresis {#hysteresis}
+To convey to EigenLayer that an EigenPod has validator's restaked on it, anyone can submit a proof against a beacon chain state root the proves that a validator has their withdrawal credentials pointed to the pod.  The proof is verified and the EigenPod calls the EigenPodMananger that calls the StrategyManager which records the validators proven balance run through the hysteresis function worth of ETH in the "beaconChainETH" strategy.  Each EigenPod keeps track of all of the validators by the hash of their public key. For each validator, their validator index and current balance in EigenLayer is kept track of.
 
+### Proofs of Validator Balance Updates
+EigenLayer pessimistically assumes the validator has less ETH that they actually have restaked in order for the protocol to have an accurate view of the validator's restaked assets even in the case of an uncorrelated slashing event, for which the penalty is >=1 ETH.
+In the case that a validator's balance drops close to or below what is noted in EigenLayer, AVSs need to be notified of that ASAP, in order to get an accurate view of their security. 
+In the case that a validator's balance, when run through the hysteresis function, is lower or higher than what is restaked on EigenLayer, anyone is allowed to permissionlessly prove that the balance of a certain validator. If the proof is valid, the StrategyManager decrements the pod owners beacon chain ETH shares by however much is staked on EigenLayer and adds the new proven stake, i.e., the strategyManager's view of the staker's shares is an accurate representation of the consensys layer as long as timely balance update proofs are submitted.  
```

Comment:
typo (awkward working): `anyone is allowed to permissionlessly prove that the balance of a certain validator`

perhaps just change to  `anyone is allowed to permissionlessly prove the new balance of the validator, triggering an update in EigenLayer` ?

Reply:
also typo: `consensys` => `consensus`

Reply:
fixed